### ISSUE TITLE
Is there a way to customize web interface?

### ISSUE THREAD

**Issue by TunaFish111** [spacehuhn/esp8266_deauther#1747]
Hello im kinda new to this stuff. But is there a way to change the web colors? Its totally fine if i cant but it would be really cool if i could make the whole web interface red or smtn. Thanks!

**Comment by IDknoobcoder** [comment]
Yeah you could go to web interface folder if u know how to code in html and css also javascript u could easily do that

**Comment by TunaFish111** [comment]
> Yeah you could go to web interface folder if u know how to code in html and css also javascript u could easily do that

I changed some stuff in the html and css code. I put it in the .gz files but when i run it its always the default web interface please help.

**Comment by IDknoobcoder** [comment]
Pls tell path of gz file

**Comment by TunaFish111** [comment]
> Pls tell path of gz file

It was in deauthner\data\web pretty sure im not home rn

**Comment by IDknoobcoder** [comment]
Wht r u doing like changing the code then compressing it to gz file I think we need to code the web interface files then change or compress all those files and save in gz folder

**Comment by TunaFish111** [comment]
> Wht r u doing like changing the code then compressing it to gz file I think we need to code the web interface files then change or compress all those files and save in gz folder

I changed the web interface files. What do i do now? How do i compress them?

**Comment by IDknoobcoder** [comment]
Maybe use 7zip or try winrar

**Comment by IDknoobcoder** [comment]
Also could u help me in my issue #1746

**Comment by TunaFish111** [comment]
> Maybe use 7zip or try winrar

What i did was customize the html and css code, save it and then compress it using 7zip and then change the files and upload it in the esp. I did this prior to these messages and it didnt seem to work. Please tell me what i did wrong.

**Comment by IDknoobcoder** [comment]
Some error messages if u could tell

**Comment by TunaFish111** [comment]
> Some error messages if u could tell

Will you be online in around 30 mins? Im heading home rn. Ill try again. However im pretty sure there wasnt any error messages. I uploaded the code everything went fine but the interface wouldnt change. Itd be great if you could try for yourself and tell me. Thanks

**Comment by IDknoobcoder** [comment]
Ok

**Comment by TunaFish111** [comment]
> Ok

Any chance you could try and tell me how you did it? Im sorry for causing you trouble

**Comment by IDknoobcoder** [comment]
Nah I can’t right now

**Comment by TunaFish111** [comment]
> Nah I can’t right now

Thats fine. So i should just compress the html files, and then change them with the ones in the web file?

**Comment by IDknoobcoder** [comment]
Yeah change the files that u made (custom version compressed) in all the gz folder hope it would help u

**Comment by TunaFish111** [comment]
> Yeah change the files that u made (custom version compressed) in all the gz folder hope it would help u

Thanks dude and sorry for wasting your time

**Comment by IDknoobcoder** [comment]
Nah bro don’t sorry I had time that is why replied 👊

**Comment by IDknoobcoder** [comment]
Did that work

**Comment by IDknoobcoder** [comment]
Does it work

**Comment by EurekaEffect** [comment]
> Does it work

No it doesn't, I have the same issue
